src/features/cheatcodes/pages/AppComponents.tsx

```diff
@@ -202,7 +212,7 @@ const AlignedText = styled(View)({
 })
 
 const StyledScrollView = styled(ScrollView)({
-  padding: 20,
+  padding: getSpacing(4),
```

Comment:
L'unité de spacing est 4. Tu voulais vraiment mettre 16 ?
```suggestion
  padding: getSpacing(5),
```